hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxyImpl.java

```diff
@@ -333,6 +334,7 @@ public boolean isLocked(K key) {
 
     @Override
     public Map<K, V> getAll(Set<K> keys) {
+        checkNotNull(keys, NULL_KEYS_ARE_NOT_ALLOWED);
```

Comment:
this seems a behavioral change since `CollectionUtil.isEmpty` already does that check and silently returns an empty map. But now user will see an exception.